src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/SymbolFactory.vb

```diff
@@ -49,6 +49,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             Return False
         End Function
 
+        Friend Overrides Function IsConstModifierType(moduleSymbol As PEModuleSymbol, type As TypeSymbol) As Boolean
+            ' VB doesn't deal with ref-readonly parameters or return-types.
+            Return False
```

Comment:
Don't understand why we are returning `False` here.  Even if VB doesn't deal with these typse returning `False` here still seems wrong.  If the function isn't implemented it should throw, not return the (potentially) wrong answer.